src/framework/xr/xr-view.js

```diff
@@ -101,10 +101,24 @@ class XrView {
         this._manager = manager;
         this._xrView = xrView;
 
-        if (this._manager.views.supportedColor)
+        if (this._manager.views.supportedColor) {
             this._xrCamera = this._xrView.camera;
 
-        this._updateTextureColor();
+            // color texture
+            if (this._manager.views.availableColor && this._xrCamera) {
+                this._textureColor = new Texture(this._manager.app.graphicsDevice, {
+                    format: PIXELFORMAT_RGB8,
```

Comment:
Just a quick check - is this always RGB8? Or could there be an alpha channel?

Reply:
Currently Specs [do not provide such information](https://immersive-web.github.io/raw-camera-access/#issue-63423cac) and from my testing it was the format provided.